testdata/picard/sam/summary_alignment_stats_test_chimeras.sam

```diff
@@ -0,0 +1,18 @@
+@HD	VN:1.0	SO:coordinate
+@SQ	SN:chr1	LN:101
+@SQ	SN:chr2	LN:101
+@SQ	SN:chr3	LN:101
+@SQ	SN:chr4	LN:101
+@SQ	SN:chr5	LN:101
+@SQ	SN:chr6	LN:101
+@SQ	SN:chr7	LN:202
+@SQ	SN:chr8	LN:202
+@RG	ID:0	SM:Hi,Momma!	LB:whatever	PU:me	PL:ILLUMINA
+SL-XAV:1:1:0:764#0/1	113	chr1	1	255	6M	chr1	10	255	TTCATG	&/,&-.
+SL-XAV:1:1:0:764#0/1	177	chr1	10	255	6M	chr1	1	255	TTCATG	&/,&-.
+SL-XAV:1:1:0:800#0/1	81	chr2	1	255	6M	chr2	10	255	TTCATG	&/,&-.
+SL-XAV:1:1:0:800#0/1	161	chr2	10	255	6M	chr2	1	255	TTCATG	&/,&-.
+SL-XAV:1:1:0:877#0/1	81	chr3	1	255	6M	chr4	10	255	TTCATG	&/,&-.
+SL-XAV:1:1:0:877#0/1	161	chr4	10	255	6M	chr3	1	255	TTCATG	&/,&-.
+SL-XAV:1:1:0:990#0/1	97	chr5	1	255	6M	chr5	10	255	TTCATG	&/,&-.
+SL-XAV:1:1:0:990#0/1	145	chr5	10	255	6M	chr5	1	255	TTCATG	&/,&-.
```

Comment:
this is only testing the orientation part of the if statement, not the insert-size, not the SA tag, not the reference sequence....